packages/now-build-utils/src/fs/run-user-scripts.ts

```diff
@@ -183,6 +187,23 @@ export async function runNpmInstall(
       opts
     );
   }
+
+  if (packageJson) {
+    // Add `node_modules/.bin` to the PATH env
+    const { stdout } = hasPackageLockJson
+      ? await execa('npm', ['bin'], { cwd: destPath })
+      : await execa('yarn', ['bin'], { cwd: destPath });
+
+    process.env.PATH = `${stdout.trim()}${path.delimiter}${process.env.PATH}`;
```

Comment:
I'm not sure this is correct because it will modify the PATH of everything including the builder itself.

Typically, we use `spawnOptions` to pass the PATH to subsequent calls.

Reply:
Yup, makes sense, adjusted it.

I also think it makes more sense to only do it inside of `@now/build-utils`, since other builders won't use this anyways.